Plugin.pm

```diff
@@ -41,6 +41,10 @@ if ($canLossless) {
 	$canLossless = 0 if $@;
 }
 
+$prefs->init({
+	replayGain => 0
```

Comment:
Could you please add `showInRadioMenu => 0` while you're at it?

Reply:
Sure.

Reply:
Done.